### PR TITLE
MET-5154 Handle Single Processed Record

### PR DESCRIPTION
**Pull request by andyjmaclean**

### CODE REVIEW

File: projects/metis/src/app/dataset/datasetlog/datasetlog.component.ts [hunk 1]
```diff
@@ -154,6 +154,8 @@ export class DatasetlogComponent extends DataPollingComponent implements OnInit
     if (log) {
       this.noLogMessage = undefined;
     } else if (missingProcessedRecords) {
+      console.log('yes we are here...');
```
Comment (jeortizquan) [inline projects/metis/src/app/dataset/datasetlog/datasetlog.component.ts:157]: clean up code

**Review by jeortizquan — COMMENTED** [review]
It works as expected just clean up the code...
afbeelding

File: projects/metis/src/app/dataset/datasetlog/datasetlog.component.ts (hunk 1 above)
Comment (andyjmaclean) [reply projects/metis/src/app/dataset/datasetlog/datasetlog.component.ts]: Fixed

File: projects/metis/src/app/dataset/datasetlog/datasetlog.component.ts [hunk 2]
```diff
@@ -165,6 +167,15 @@ export class DatasetlogComponent extends DataPollingComponent implements OnInit
...
   */
   getLogFrom(logTo: number): number {
-    return logTo - this.logPerStep >= 1 ? logTo - this.logPerStep + 1 : 1;
+    let res = 1;
+
+    if (logTo - this.logPerStep >= 1) {
+      res = logTo - this.logPerStep + 1;
+    }
+
+    console.log('get log from (' + logTo + ') returns ' + res);
```
[the remark this answers is not in the document]
Comment (andyjmaclean) [reply projects/metis/src/app/dataset/datasetlog/datasetlog.component.ts]: Fixed

**Pull request merged**